src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -63,11 +63,17 @@ export const convertResult = (body: IDataFrame) => {
   return hits;
 };
 
+enum FeedbackStatus {
+  NONE,
+  THUMB_UP,
+  THUMB_DOWN,
+}
+
 export const QueryAssistSummary: React.FC<QueryAssistSummaryProps> = (props) => {
   const { query, search } = props.data;
-  const [summary, setSummary] = useState(null); // store fetched data
```

Comment:
In case LLM fails to generate summary, we would render an error message. But if LLM is able to generate summary, I believe there's always some outputs.